65black/StackOverflow-lite
# #160370596 Get User Details

## What does this PR do?
Adds logic for the` GET /api/v1/users` endpoint.
The user must be signed in to access this endpoint.

If the user is signed in, then hitting this endpoint will return all the details of the user.

### Description of Task to be completed?
Have the following endpoints working:

> GET /api/v1/users

### How should this be manually tested?
After cloning the repo, cd into it and run `npm run start-dev`. Follow these steps:

 - Open postman and hit the login endpoint first to get your auth token.
 - Add the token to your request header and hit `/api/v1/users`.
 - You should get a response with all your details

### Screenshots
![scr](https://user-images.githubusercontent.com/36507972/45293329-c5972300-b4ef-11e8-9cfb-e6749db066fd.JPG)

### What are the relevant pivotal tracker stories?
#160370596
